account_edi_simple_pdf/README.rst

```diff
@@ -0,0 +1,212 @@
+
+You can choose one extraction method and only install the tools/libs for that method.
+
+Install PyMuPDF
+~~~~~~~~~~~~~~~
+
+Install it via pip:
+
+.. code::
+
+  pip3 install --upgrade pymupdf
+
+Beware that *PyMuPDF* is not a pure-python library: it uses MuPDF, which is written in C language. If a python wheel for your OS, CPU architecture and Python version is available on pypi (check the `list of PyMuPDF wheels <https://pypi.org/project/PyMuPDF/#files>`_ on pypi), it will install smoothly. Otherwize, the installation via pip will require MuPDF and all its development libs to compile the binding.
+
+Install pdftotext python lib
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install **pdftotext python lib**, run:
+
+.. code::
+
+  sudo apt install build-essential libpoppler-cpp-dev pkg-config python3-dev
+
+and then install the lib via pip:
+
+.. code::
+
+  pip3 install --upgrade pdftotext
+
+On OSes other than Debian/Ubuntu, follow the instructions on the `project page <https://github.com/jalan/pdftotext>`_.
+
+Install pdftotext command line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install **pdftotext command line**, run:
+
+.. code::
+
+  sudo apt install poppler-utils
+
+Install pypdf
+~~~~~~~~~~~~~
+
+To install the **pypdf** python lib, run:
+
+.. code::
+
+  pip3 install --upgrade pypdf
+
+
+Other requirements
+~~~~~~~~~~~~~~~~~~
+
+This module also requires the following Python libraries:
+
+* `regex <https://pypi.org/project/regex/>`_ which is backward-compatible with the *re* module of the Python standard library, but has additional functionalities.
+* `dateparser <https://github.com/scrapinghub/dateparser>`_ which is a powerful date parsing library.
+
+The dateparser lib depends itself on regex. So you can install these Python libraries via pip with the following command:
+
+.. code::
+
+  pip3 install --upgrade dateparser
+
+The dateparser lib is not compatible with all regex lib versions. As of February 2024, the `version requirement <https://github.com/scrapinghub/dateparser/blob/master/setup.py#L36>`_ declared by dateparser for regex is **!=2019.02.19, !=2021.8.27**. So the latest version of dateparser is currenly compatible with the latest version of regex. To know the version of regex installed in your environment, run:
+
+
+.. code::
+
+  pip3 show regex
+
+Configuration
+=============
+
+By default, for the PDF to text conversion, the module tries the different methods in the order mentioned in the INSTALL section: it will first try to use **PyMuPDF**; if it fails (for example because the lib is not properly installed), then it will try to use the **pdftotext python lib**, if that one also fails, it will try to use **pdftotext command line** and, if it also fails, it will eventually try **pypdf**. If none of the 4 methods work, Odoo will display an error message.
```

Comment:
I assume this try-and-error refers to the general availability of the different methods but not to their successful extraction on each individual PDF file. Would be good to clarify this.